staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go

```diff
@@ -91,18 +92,30 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	result := make(chan interface{})
```

Comment:
using context here might be better @sttts wdyt?

Reply:
sending a struct to a channel seems cleaner to me